### ISSUE TITLE
Training with pretrained model

### ISSUE THREAD

**Issue by ilpapds** [ultralytics/ultralytics#1441]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

After training yolov8s from scratch for 200 epochs, I got mAP@50=69.0% . However, after trying to train it further for another 100 epochs with this command:

yolo task=detect mode=train model=runs/detect/yolov8s_bdd__3_cls/weights/last.pt  imgsz=640 data=datasets/bdd.yaml epochs=100 batch=24 name=yolov8s_bdd__3_cls pretrained=True

However, mAP@50 at first epoch (theoretically 201 epoch) is 67.1%. Did i make a mistake while using my pretrained model file? Thank you in advance

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@ilpapds this is expected. Do not raise bug reports for questions.

LR schedulers drop to near zero on final epoch for any training. Starting a new training returns the scheduler to it's original value.
